spec/services/checksum_validator_spec.rb

```diff
@@ -315,5 +315,22 @@
       expect(cv).to receive(:flag_unexpected_data_files)
       cv.validate_signature_catalog
     end
+
+    it 'rescues if the file or directory does not exist' do
```

Comment:
`adds error code and continues executing if a file or directory does not exist` ??